scss/bootstrap-prod.scss

```diff
@@ -5,45 +5,9 @@
  */
 
 .bs {
+    @import "../node_modules/bootstrap/scss/bootstrap";
```

Comment:
@mwiencek Is this alright? I reckon we would want to ship bootstrap to all our projects from here directly and hence decided to do this. Let me know if this seems alright to you.

Reply:
I agree we should look to minimize the size of the CSS. I will look to import only useful individual modules now!
Also, I would like us to maintain the respective CSS here and Import it to our projects directly if that's a good idea.

Reply:
> Include this before custom-vars works fine btw!

That doesn't appear to be strictly true; check the diff for dist/bootstrap.css where e.g. the colors defined at the top are wrong now, whereas they were correct before.